great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -790,6 +810,31 @@ def add_checkpoint(
         self.checkpoint_store.add_checkpoint(checkpoint, name, ge_cloud_id)
         return checkpoint
 
+    def get_checkpoint(
+        self,
+        name: Optional[str] = None,
+        ge_cloud_id: Optional[str] = None,
+    ) -> Checkpoint:
+        checkpoint_config: CheckpointConfig = self.checkpoint_store.get_checkpoint(
+            name=name, ge_cloud_id=ge_cloud_id
+        )
+        checkpoint: Checkpoint = Checkpoint.instantiate_from_config_with_runtime_args(
+            checkpoint_config=checkpoint_config,
+            data_context=self,
+            name=name,
+        )
+
+        return checkpoint
+
+    def delete_checkpoint(
+        self,
+        name: Optional[str] = None,
```

Comment:
This seems like it should be required right?

Reply:
so it's essentially an XOR - one needs to be passed in. I agree that it is confusing but since the nature of this PR is simply porting over the methods to the parent class, I think we should tackle this matter in a subsequent ticket.